TechnologyAssessmentApp/TechnologyAssessmentApp/POP Pattern/ShakeDeviceProtocol.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  ShakeDeviceProtocol.swift
+//  TechnologyAssessmentApp
+//
+//  Created by Kshitij Godara on 25/08/18.
+//  Copyright © 2018 Nagarro. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+protocol ShakeDeviceProtocol { }
```

Comment:
I don't understand the naming of `ShakeDeviceProtocol`. Maybe you are missing comments or your created 2 protocol for the same purpose ?